docs/api.md

```diff
@@ -3353,7 +3353,7 @@ __Examples__
 Ractive.sharedSet( '_', lodash );
 ```
 
-## Ractive.styleSet()
+## <a name="styleset"></a>Ractive.styleSet()
```

Comment:
MkDocs automatically generates linked headers based on the header text.

Reply:
Argh, sorry, mistake. I'll redo it.